src/components/profilePhotos/UploadDialog.jsx

```diff
@@ -4,8 +4,7 @@ import { FormattedMessage } from 'react-intl';
 import 'react-image-crop/dist/ReactCrop.css';
 import DialogActions from '@material-ui/core/DialogActions';
 import DialogContent from '@material-ui/core/DialogContent';
-import Alert from '@material-ui/lab/Alert';
-import AlertTitle from '@material-ui/lab/AlertTitle';
+import CustomAlert from '../Alert';
```

Comment:
not important, but why call it CustomAlert everywhere?

Reply:
Future-proofing? I figured that someone coming into the code base might take a moment before realizing that our <Alert> was a custom wrapper. So why not just call it that everywhere?

Reply:
Sounds good to me!